### ISSUE TITLE
Write some e2e tests for `FAQ` page to avoid regressions

### ISSUE THREAD

**Issue by zainfathoni** [kawalcovid19/wargabantuwarga.com#380]

**Comment by afifalfiano** [comment]
Hello @zainfathoni mas, Can I know the limitations of doing e2e on the faq page?

**Comment by zainfathoni** [comment]
> Hello @zainfathoni mas, Can I know the limitations of doing e2e on the faq page?

I don't think we have any significant limitations. The page should be completely testable using Cypress without any constraints or limitations, since it's a statically-built public page.

**Comment by afifalfiano** [comment]
> > Hello @zainfathoni mas, Can I know the limitations of doing e2e on the faq page?
> 
> I don't think we have any significant limitations. The page should be completely testable using Cypress without any constraints or limitations, since it's a statically-built public page.

Okee, can I handle this issue?

**Comment by zainfathoni** [comment]
> Okee, can I handle this issue?

Sure, no need to ask permissions. Just check if there's any other PR linked to this issue. If there is no PR linked, just start working on it and create a [draft pull request](https://kcov.id/draft-pr) as soon as possible to claim this issue.

**Comment by afifalfiano** [comment]
Okay mas, if possible, please review my draft PR for this issue. thank you

> > Okee, can I handle this issue?
> 
> Sure, no need to ask permissions. Just check if there's any other PR linked to this issue. If there is no PR linked, just start working on it and create a [draft pull request](https://kcov.id/draft-pr) as soon as possible to claim this issue.

**Comment by zainfathoni** [comment]
I'll review your PR once it's marked as ready for review.

**Comment by afifalfiano** [comment]
> I'll review your PR once it's marked as ready for review.

Okee, thank mas
